code/.env.sample

```diff
@@ -1,5 +1,16 @@
-ETH_GANACHE_KEY=60537f2b5c5fa6180d3b65c181f005828ddb2799a2d776cf7c5f5ce1935a2f8a
+ETH_GANACHE_MNEMONIC='filter group there hunt fitness junior ghost park route jar entire clown allow rifle meadow'
```

Comment:
new config from master supports mnemonic and no longer needs multiple keys

Reply:
Have removed the keys.
At some point, if needed, we can revisit how we create accounts that can sign.
Currently have left the `ETH_GANACHE_MNEMONIC` in as I use it for testing.
🙏